### DISCUSSION TITLE
glTF Working Group Meeting 2021-12-09

### DISCUSSION THREAD

**Discussion by fire** [omigroup/gltf-extensions#40] [General]
## Agenda

- Welcome newcomers.
- Khronos Liaison Update.
- OMI_audio_emitter updates
- Schedule working sessions for explainer and blender Addon

## Addons

* Future extensions:

**Comment by fire** [comment]
Fire and SingerGirl collaborated over Discord messages for the OMI-GLTF-WG over the week of 2021-12-06

## Omi audio emitter 

\<SingerGirl> Having a bird model would be good as a test. We need to find a free recording of that birdsong.

\<SingerGirl> It would be great to do a bird sanctuary where you heard songs only from a certain distance. Could teach people about the different species.

## Procedural apparel on GLTF2.

This is used for procedural gltf2 through MaterialX. Nothing found yet.

## Animation on texture UVs for a lava volcano

\<SingerGirl> [volcano.zip (Blend)](https://github.com/omigroup/gltf-extensions/files/7688732/volcano.zip)

**Comment by robertlong** [comment]
# Notes

- Discussion on `https://` and other URI's with specified protocols and support within the Blender glTF Exporter
  - Concern on previewing these in Blender
  - URIs can make assets more brittle than a self-contained glb
  - The glTF spec does specify that URI protocols can be specified but that clients must support relative paths and data uris https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#uris
- VCI (virtual-cast) glTF extension ecosystem
  - https://github.com/virtual-cast/VCI/tree/master/Assets/VCI/UniVCI/Scripts/Format/Serializer
  - Possibly a good partner to work with on glTF extensions or to adopt these as-is within OMI
- Evie to look into creating a standard bird model for demoing the audio emitter

**Comment by EvieMarie** [comment]
[tanager.zip](https://github.com/omigroup/gltf-extensions/files/7689138/tanager.zip)
Scarlet Tanager Bird - separated GLTF with audio file for test
I created it.
License:  CC0 or CCBY
